server/survey.go

```diff
@@ -45,7 +45,7 @@ func (p *Plugin) checkForNextSurvey(currentVersion semver.Version) bool {
 		return false
 	}
 
-	now := time.Now()
+	now := time.Now().UTC()
```

Comment:
@hmhealey added the UTC to make the time more standard. to avoid timezones

Reply:
Good call